vue-components/src/components/Icon.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+	<!-- eslint-disable max-len -->
+	<span :class="['wikit', 'wikit-Icon', `wikit-Icon--${size}`, `wikit-Icon--${color}`]">
+		<!-- error icon -->
+		<svg
+			class="wikit-Icon__svg"
+			viewBox="0 0 16 16"
+			fill="none"
+			xmlns="http://www.w3.org/2000/svg"
+			v-if="type === 'error'"
+		>
+			<path fill="currentColor" d="M11.4939 0H4.50612L0 4.50612V11.5102L4.50612 16H11.5102L16 11.4939V4.50612L11.4939 0Z" />
+			<path
+				fill-rule="evenodd"
+				clip-rule="evenodd"
+				d="M8.8 11.9938H7.2V10.3999H8.8V11.9938Z"
+				fill="#FEE7E6"
```

Comment:
The color seems wrong, shouldn't be "currentColor"?

Reply:
So this path is actually the little exclamation mark within the icon outline, so it shouldn't be `currentColor`, but you're right that this fill color is out of place. @guergana and I talked about this with @SaiSan-WMDE earlier and agreed that we will replace the entire icon soon. This, as you see it now, is how we copied it out of Figma, but we actually want this exclamation mark to be transparent, not white-pink-ish(?!).

Reply:
ack!

Reply:
The latest update is an attempt at making the exclamation mark transparent. (I should've just done that sooner, sorry)